nix/cabal-project.nix

```diff
@@ -32,14 +32,13 @@ let
   };
 
   # TODO this is temporary and will be done automatically by IOGX in the next version
-  cardano-node-gitrev = "eb66a54f5d03d96fa9f4899b4f43ebf4e829f2ca";
+  cardano-node-gitrev = "a29ee68ba2c850cd50be39a3105ef191cfbc41d5";
 
   shellWithHoogle = false;
 
   sha256map = {
-    "https://github.com/input-output-hk/cardano-node".${cardano-node-gitrev} = "sha256-QYqWuJPk0frnBrcRAv4SzvAckLMBnaKThP5Gl5lHN44=";
```

Comment:
Maybe you don't want to delete this? (reference to `cardano-node-gitrev` has disappeared)

Reply:
It doesn't seem to be useful , so I removed it. I am no longer overriding cardano-node repo. The correct executables are being used and they display correct version. It is still being used to `setGitRevForPaths`